lib/pdf_fill/forms/va21p0969.rb

```diff
@@ -132,6 +237,18 @@ def expand_unassociated_income(income)
         }
       end
 
+      def get_middle_initial(middle_name)
```

Comment:
Same.. comments, please

##
    # Does this thing because XYZ
    #
    # @param [String] foo
    #
    # @return [String] the return type is a cool thing
    #